pkg/cmd/experimental/haconfig/haconfig.go

```diff
@@ -0,0 +1,188 @@
+package haconfig
+
+import (
+	"fmt"
+	"io"
+
+	cmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"github.com/openshift/origin/pkg/cmd/util/variable"
+	hac "github.com/openshift/origin/pkg/haconfig"
+	"github.com/openshift/origin/plugins/haconfig/keepalived"
+)
+
+const shortDesc = "Configure or view High Availability configuration"
+const description = `
+Configure or view High Availability configuration
+
+This command helps to setup High Availability (HA) configuration for an
+OpenShift environment. An administrator can configure HA on an entire
+cluster or as would normally be the case on a subset of nodes (as defined
+via a labelled selector).
+If no arguments are passed, this command will display the HA configuration
+for a resource name 'ha-config'.
+
+If a HA configuration does not exist with the given name, the --create flag
+can be passed to create a deployment configuration and service that will
+provide HA and failover capability. If you are running in production, it is
+recommended that the labelled selector for the nodes matches atleast 2
+nodes to ensure you have failover protection and that you provide a
+--replicas=<n> value that matches the number of nodes for the given
+labelled selector.
+
+
+Examples:
+  Check the default HA configuration ("ha-config"):
+
+  $ %[1]s %[2]s
+
+  See what the HA configuration would look like if it is created:
+
+  $ %[1]s %[2]s -o json
+
+  Create a HA configuration if it does not already exist:
+
+  $ %[1]s %[2]s hac --virtual-ips="10.1.1.1-4" --create
+
+  Create a HA configuration on a selection of nodes labelled
+  "router=us-west-ha" (on 4 nodes with 7 virtual IPs monitoring a service
+  listening on port 80 (aka the OpenShift router process).
+
+  $ %[1]s %[2]s ha-config --selector="router=us-west-ha" --virtual-ips="1.2.3.4,10.1.1.100-104,5.6.7.8" --watch-port=80 --replicas=4 --create
+
+  Delete a previously created HA configuration:
+
+  $ %[1]s %[2]s hac --delete
```

Comment:
Don't need this, we have the "stop" command and delete for this.  Just ensure you have a default label setup.